internal/stacks/fireflyIdentity.go

```diff
@@ -86,14 +88,16 @@ func (s *Stack) httpJSON(method, url string, body, result interface{}) (err erro
 }
 
 func (s *Stack) registerFireflyIdentities(spin *spinner.Spinner, verbose bool) error {
+	emptyObject := make(map[string]interface{})
+
 	for _, member := range s.Members {
 		orgName := fmt.Sprintf("org_%s", member.ID)
 		nodeName := fmt.Sprintf("node_%s", member.ID)
 		ffURL := fmt.Sprintf("http://127.0.0.1:%d/api/v1", member.ExposedFireflyPort)
 		updateStatus(fmt.Sprintf("registering %s and %s", orgName, nodeName), spin)
 
 		registerOrgURL := fmt.Sprintf("%s/network/register/node/organization", ffURL)
-		err := s.httpJSONWithRetry(http.MethodPost, registerOrgURL, nil, nil)
+		err := s.httpJSONWithRetry(http.MethodPost, registerOrgURL, emptyObject, nil)
```

Comment:
I find it weird that this body is required, so I opened a bug on FireFly: https://github.com/hyperledger-labs/firefly/issues/155